embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/opentelemetry/OpenTelemetrySdk.kt

```diff
@@ -0,0 +1,32 @@
+package io.embrace.android.embracesdk.opentelemetry
+
+import io.embrace.android.embracesdk.BuildConfig
+import io.opentelemetry.api.trace.Tracer
+import io.opentelemetry.sdk.OpenTelemetrySdk
+import io.opentelemetry.sdk.common.Clock
+import io.opentelemetry.sdk.trace.SdkTracerProvider
+import io.opentelemetry.sdk.trace.SpanProcessor
+
+/**
+ * Wrapper that instantiates a copy of the OpenTelemetry SDK configured with the appropriate settings and the given components so
+ * the Embrace SDK can hook into its lifecycle. From this, the Embrace SDK can obtain an implementations of the OpenTelemetry API to
+ * create OpenTelemetry primitives that it can use internally or export to any OpenTelemetry Collectors.
+ */
+internal class OpenTelemetrySdk(
+    openTelemetryClock: Clock,
+    spanProcessor: SpanProcessor
+) {
+    private val sdk = OpenTelemetrySdk
+        .builder()
```

Comment:
@lucaslabari we can set a `SdkLoggerProvider` here and add a method to return the logger instance

Reply:
sound good !!!